DigitalLearningSolutions.Web/Helpers/SelectListHelper.cs

```diff
@@ -25,5 +28,11 @@ public static IEnumerable<SelectListItem> MapOptionsToSelectListItems(IEnumerabl
         {
             return options.Select(option => new SelectListItem(option.Text, option.Value));
         }
+
+        public static IEnumerable<SelectListItem> MapPeriodOptionsToSelectListItem(int? selectedId = null)
+        {
+            var values = Enumeration.GetAll<Period>();
+            return values.Select(v => new SelectListItem(v.Name, v.Id.ToString(), v.Id == selectedId));
+        }

```

Comment:
This seems a bit too specific for this helper.. I don't think we'll be reusing it?